RoddyCore/src/de/dkfz/roddy/config/ConfigurationValue.groovy

```diff
@@ -119,7 +121,7 @@ private String checkAndCorrectPathThrow(String temp) throws ConfigurationError {
         String curUserPath = (new File("")).getAbsolutePath();
         String applicationDirectory = Roddy.getApplicationDirectory().getAbsolutePath();
         //TODO Make something like a blacklist. This is not properly handled now. Initially this was done because Java sometimes puts something in front of the file paths.
-        if (value.startsWith("${") || value.startsWith("$") || value.startsWith("~") || !value.startsWith("/")) {
+        if (value.startsWith('${') || value.startsWith('$') || value.startsWith("~") || !value.startsWith("/")) {
```

Comment:
Now that you touched that code, would it not make sense generalizing it and moving it into e.g. a dedicated class. From there we could evolve it into a true templating system, or use an existing templating system. We use similar code also for the variables in the filename patterns. Also, there are plenty of repetitions here -- '${', etc.

Reply:
It would. But not for now, that'll be a bigger discussion right?

Reply:
Right.